packages/logger/src/Logger.ts

```diff
@@ -69,48 +105,120 @@ class Logger implements ClassThatLogs {
   }
 
+  /**
+   * Alias for addPersistentLogAttributes.
+   *
+   * @param {LogAttributes} attributes
+   * @returns {void}
+   */
   public appendKeys(attributes?: LogAttributes): void {
     this.addPersistentLogAttributes(attributes);
   }
 
+  /**
+   * it creates a separate Logger instance, identical to the current one.
+   * It's possible to overwrite the new instance options by passing them.
+   *
+   * @param {LoggerOptions} options
+   * @returns {Logger}
+   */
   public createChild(options: LoggerOptions = {}): Logger {
     return cloneDeep(this).setOptions(options);
   }
 
+  /**
+   * It prints a log item with level DEBUG.
+   *
+   * @param {LogItemMessage} input
+   * @param {Error | LogAttributes | unknown} extraInput
+   * @returns {void}
+   */
   public debug(input: LogItemMessage, ...extraInput: LogItemExtraInput): void {
     this.processLogItem('DEBUG', input, extraInput);
   }
 
+  /**
+   * It prints a log item with level ERROR.
+   *
+   * @param {LogItemMessage} input
+   * @param {Error | LogAttributes | unknown} extraInput
+   * @returns {void}
+   */
   public error(input: LogItemMessage, ...extraInput: LogItemExtraInput): void {
     this.processLogItem('ERROR', input, extraInput);
   }
 
+  /**
+   * It evaluates whether the current Lambda function invocation has a cold start or not.
+   *
+   * @static
+   * @returns {void}
+   */
   public static evaluateColdStartOnce(): void {
     if (!Logger.getColdStartEvaluatedValue()) {
       Logger.evaluateColdStart();
     }
   }
 
+  /**
+   * It returns a boolean value which is true if the current Lambda function cold start has been already evaluated, false if not.
+   *
+   * @static
+   * @returns {boolean}
+   */
   public static getColdStartEvaluatedValue(): boolean {
     return Logger.coldStartEvaluated;
   }
 
+  /**
+   * It returns an optional boolean value, true if the current Lambda function invocation has a cold start, false otherwise.
+   *
+   * @static
+   * @returns {boolean | undefined}
+   */
   public static getColdStartValue(): boolean | undefined {
     return Logger.coldStart;
   }
 
+  /**
+   * It returns a boolean value, if true all the logs will be printed.
+   *
+   * @returns {boolean}
+   */
   public getLogsSampled(): boolean {
     return this.logsSampled;
   }
 
+  /**
+   * It prints a log item with level INFO.
+   *
+   * @param {LogItemMessage} input
+   * @param {Error | LogAttributes | unknown} extraInput
+   * @returns {void}
+   */
   public info(input: LogItemMessage, ...extraInput: LogItemExtraInput): void {
     this.processLogItem('INFO', input, extraInput);
   }
 
+  /**
+   * Method decorator that adds the current Lambda function context as extra
+   * information in all log items.
+   * The decorator can be used only when attached to a Lambda function handler which
+   * is written as method of a class, and should be declared just before the handler declaration.
+   *
```

Comment:
Since this is a slightly more involved method perhaps we can provide a code example?

![image](https://user-images.githubusercontent.com/7353869/150309077-e123db38-c3dd-49bd-b7a2-47d4cfb5b6ac.png)